addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -333,6 +334,44 @@ describe('Paste', () => {
                     contentAfter: '<ul><li><h1>Google</h1></li><li><h1>Test</h1></li><li><h1>test2</h1></li></ul><p>[]<br></p>',
                 });
             });
+            it('should convert p into span if p has classes', async () => {
```

Comment:
1. Can you move this unit test from copy paste to sanitize please.
Your change is in `sanitize.js` so it do not make sense to test it with a unit test in `copyPaste.test.js`.

2. Your unit test is way too complicated, you should always write the smallest unit test possible that actuality test your change.
Something like : 

```
...
contentBefore: '<ul><li><p class="class-1">abc</p><p class="class-2">def</p></li></ul>',
contentAfter: '<ul><li><span class="class-1">abc</span><span class="class-2">def</span></li></ul>'
...
```

Reply:
@sebgeelen Done changes. :+1: